src/signals/incident/definitions/wizard-step-1-beschrijf.js

```diff
@@ -33,6 +33,7 @@ const getControls = memoize(
           label: 'Waar is het?',
           subtitle: 'Typ het dichtstbijzijnde adres of klik de locatie aan op de kaart',
           path: 'location',
+          requiredErrorMessage: 'Kies een locatie op de kaart of vul een adres in',
```

Comment:
This construction is too specific. Please use either an custom validator here (`src/signals/incident/services/custom-validators/index.js`) or an override of the `getError` meta function?

Reply:
I started out with a custom validator, but got stuck - I gave it another try and have got a working version with custom validator (see last commit)